src/fullnode.rs

```diff
@@ -182,21 +183,7 @@ impl Fullnode {
             sigverify_disabled,
             rpc_port,
             storage_rotate_count,
-        );
-
-        match leader_addr {
-            Some(leader_addr) => {
-                info!(
-                    "validator ready... rpc address: {}, connected to: {}",
-                    rpc_addr, leader_addr
-                );
-            }
-            None => {
-                info!("leader ready... rpc address: {}", rpc_addr);
```

Comment:
>  It will print "leader ready..." when it's ready to receive transactions.

https://github.com/solana-labs/solana/blob/978b26a9c/docs/src/cluster/bench-tps.md